src/category_theory/isomorphism.lean

```diff
@@ -236,6 +236,7 @@ variables {f g : X ⟶ Y} {h : Y ⟶ Z}
 instance inv_is_iso [is_iso f] : is_iso (inv f) :=
 is_iso.of_iso_inv (as_iso f)
 
+@[priority 900]
```

Comment:
```suggestion
/- The following instance has lower priority for the following reason:
Given `f : X ≅ Y` with `X Y : Type u`, without this priority change, typeclass inference cannot deduce `is_iso f.hom` because `f.hom` is defeq to `(λ x, x) ≫ f.hom`, triggering a loop. -/
@[priority 900]
```